client/src/app/tabs/form/FormEditor.js

```diff
@@ -227,6 +235,23 @@ export class FormEditor extends CachedComponent {
     }
   }
 
+  handleLinting = () => {
+    const {
+      engineProfile,
+      form
+    } = this.getCached();
+
+    if (!engineProfile) {
+      return;
+    }
+
+    const contents = form.getSchema();
+
+    const { onAction } = this.props;
+
+    onAction('lint-tab', { contents });

```

Comment:
Why don't we make this a generic `changed` event? What is linting specific here except for the `engineProfile`?

Reply:
We've talked about this: There is already two different _changed_ things in our app (`onChanged`, `onContentUpdated`). Adding a third one does not do too much harm.